pages/topics/gateway-events.mdx

```diff
@@ -1865,6 +1983,131 @@ Sent when a [stage instance](/resources/stage-instance) is updated. The inner pa
 
 Sent when a [stage instance](/resources/stage-instance) is deleted (i.e. the stage has been closed). The inner payload is a [stage instance](/resources/stage-instance#stage-instance-object) object.
 
+### Streams
+
+#### Stream Object
+
+###### Stream Structure
+
+| Field             | Type      | Description                                                            |
+| ----------------- | --------- | ---------------------------------------------------------------------- |
+| stream_key        | string    | The [stream key](#stream-key) for the stream                           |
+| rtc_server_id ^1^ | snowflake | The ID of the RTC server for the stream, used when connecting to voice |
+| region            | string    | The region of the stream                                               |
```

Comment:
```suggestion
| region            | string    | The voice region the stream is in                                               |
```

redundant, should have more information ("the region the client must change to"?)